MIDIchlorians/MIDIchlorians/Animation.swift

```diff
@@ -12,19 +12,16 @@ class Animation: Object {
 
     private dynamic var animationString: String?
 
-    convenience init(_ animation: AnimationSequence) {
+    convenience init(_ animationType: String) {
```

Comment:
No conflicts with #35 since the functions in this class are not getting called anywhere yet.
Once this gets merged, I'll add code to create and retrieve animations in `AnimationManager` (should be one-liners).